.github/workflows/call-build-windows.yaml

```diff
@@ -45,14 +45,17 @@ jobs:
         config:
           - name: "Windows 32bit"
             arch: x86
-            vcpkg_triplet: x86-windows-static
-            openssl_root_dir_opt: -DOPENSSL_ROOT_DIR=C:\vcpkg\packages\openssl_x86-windows-static
+            win32openssl: https://slproweb.com/download/Win32OpenSSL-1_1_1q.msi
+            win32openssl_dir: "C:\\Program\ Files\\OpenSSL-Win32"
           - name: "Windows 64bit"
             arch: x64
-            vcpkg_triplet: x64-windows-static
-            openssl_root_dir_opt: ""
+            win32openssl: https://slproweb.com/download/Win64OpenSSL-1_1_1q.msi
```

Comment:
we probably should just name this `openssl_dir` or something rather than `win32` in the 64-bit case...

Reply:
Yup. I used openssl_dir for this variable.